packages/x-date-pickers/src/PickersDay/PickersDay.tsx

```diff
@@ -19,11 +19,28 @@ import {
   pickersDayClasses,
 } from './pickersDayClasses';
 
+export interface ExportedPickersDayProps {
+  /**
+   * If `true`, today's date is rendering without highlighting with circle.
+   * @default false
+   */
+  disableHighlightToday?: boolean;
+  /**
+   * If `true`, days outside the current month are rendered:
+   * - if `fixedWeekNumber` is defined, renders days to have the weeks requested.
+   * - if `fixedWeekNumber` is not defined, renders day to fill the first and last week of the current month.
+   * On range pickers, it will be ignored if more than one month is rendered.
```

Comment:
Maybe use the code term rather than the doc one in the JSDoc?
```suggestion
   * On range pickers, it will be ignored if more than one `calendars` is rendered.
```